backend/org.cups.usb-quirks

```diff
@@ -77,6 +77,27 @@
 # Canon, Inc. MP210 (https://bugzilla.redhat.com/show_bug.cgi?id=847923#c53)
```

Comment:
As you can see MP210 was already in the list, I wonder how many missing models in each series are slipping by.